opensearch-operator/pkg/reconcilers/dashboards.go

```diff
@@ -84,7 +85,18 @@ func (r *DashboardsReconciler) Reconcile() (ctrl.Result, error) {
 	result.CombineErr(ctrl.SetControllerReference(r.instance, cm, r.Client.Scheme()))
 	result.Combine(r.ReconcileResource(cm, reconciler.StatePresent))
 
-	deployment := builders.NewDashboardsDeploymentForCR(r.instance, volumes, volumeMounts)
+	annotations := make(map[string]string)
+
+	if cmData, ok := cm.Data[helpers.DashboardConfigName]; ok {
+		sha1sum, err := util.GetSha1Sum([]byte(cmData))
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+
+		annotations["checksum/sha1"] = sha1sum
```

Comment:
I prefer to change this from `checksum/sha1` to `checksum/dashboards-yml` to make clear where the checksum stems from.

Reply:
Good point, ill change it to "checksum/dashboards.yml" to better match the actual filename